packages/nextjs-mf/src/default-delegate.ts

```diff
@@ -6,15 +9,29 @@ import {
```

Comment:
The variable 'pure' is initialized as an empty object and then immediately reassigned. Consider initializing it with its final value directly.